plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/register-components.ts

```diff
@@ -201,6 +201,17 @@ registerCheckoutBlock( {
 	),
 } );
 
+registerCheckoutBlock( {
+	metadata: metadata.CART_ORDER_SUMMARY_TOTALS,
+	component: lazy(
+		() =>
+			import(
+				/* webpackChunkName: "cart-blocks/order-summary-shipping" */
```

Comment:
I think this needs to be changed else two chunks will have the same name

Reply:
Good spot, missed that one 👍